riffdog/command_line.py

```diff
@@ -120,21 +121,29 @@ def main(*args):
     
     # Parse args.
     
-    parsed_args = parser.parse_args()
+    parsed_args = parser.parse_args(sys.argv[1:])
 
 
     # 2. Build config object
 
     for arg in vars(parsed_args):
         setattr(config, arg, getattr(parsed_args, arg))
-
     
+    if parsed_args.dir:
+        config.state_storage = StateStorage.FILE
+        config.state_file_locations = parsed_args.dir
+    elif parsed_args.bucket:
+        config.state_storage = StateStorage.AWS_S3
+        config.state_file_locations = parsed_args.bucket[0]
+    else:
+        #? pass? - if config file driven but if not caught by the len(config.state_file_locations ebelow
+        pass
+
     if parsed_args.bucket is not None:
```

Comment:
Is this still needed?

Reply:
Nope, chopped.

Reply:
Sorry I've had lots of issues with this review - if you mean the `if parsed_args.bucket` .. yes because we now have file support? Unless you see something I don't?

Reply:
I agree, I find comments about specific lines difficult to read - I was talking about line 139-140 because they look fairly similar to 135-137?

Reply:
In which case yes, its been chopped :D